daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/ScriptF.scala

```diff
@@ -259,6 +259,49 @@ object ScriptF {
       } yield SEAppAtomic(SEValue(continue), Array(SEValue(SOptional(optR))))
   }
 
+  final case class QueryView(
+      parties: OneAnd[Set, Party],
+      interfaceId: Identifier,
+      stackTrace: StackTrace,
+      continue: SValue,
+  ) extends Cmd {
+    override def description = "queryView"
+
+    override def execute(env: Env)(implicit
+        ec: ExecutionContext,
+        mat: Materializer,
+        esf: ExecutionSequencerFactory,
+    ): Future[SExpr] = {
+
+      def makePair(v1: SValue, v2: SValue): SValue = {
+        import com.daml.lf.language.StablePackage.DA
+        import com.daml.script.converter.Converter.record
+        record(DA.Types.assertIdentifier("Tuple2"), ("_1", v1), ("_2", v2))
+      }
+
+      for {
+        viewType <- Converter.toFuture(env.lookupInterfaceViewTy(interfaceId))
+        client <- Converter.toFuture(env.clients.getPartyParticipant(parties.head))
```

Comment:
I think this should use all the parties, since the corresponding line for `Query` has

```
        client <- Converter.toFuture(
          env.clients
            .getPartiesParticipant(parties)
        )
```

```suggestion
        client <- Converter.toFuture(env.clients.getPartiesParticipant(parties))
```

Reply:
see #15381